src/main/webapp/scripts/map.js

```diff
@@ -49,31 +69,76 @@ function initMap() {
   map.addListener("click", e => {
     placeMarkerAndPanTo(e.latLng, map);
   });
+  
   /*Add Hard-Coded Markers */
   addMarker(map);
 }
 
-/* Temporary Function to Hold Marker for Sky Garden */
 function addMarker(map) {
-  const skyGarden = {lat: 51.510881, lng: -0.083751}
-  const marker = new google.maps.Marker({
-    position: skyGarden,
+  //hardcoded marker data
+  const mildredsKingsCross = new google.maps.Marker({
+    position: {lat: 51.531299, lng: -0.117168},
+    map: map,
+    title: 'Mildreds Kings Cross',
+    icon: yellowIcon
+  });
+  const skyGarden = new google.maps.Marker({
+    position: {lat: 51.510881, lng: -0.083751},
     map: map,
-    title: "Sky Garden"
+    title: 'Sky Garden',
+    icon: blueIcon
   })
+  const fitzBar = new google.maps.Marker({
+    position: {lat: 51.517735, lng: -0.097112},
+    map: map,
+    title: 'Fitz\'s Bar',
+    icon: purpleIcon
+  });
+  const bloomsburyCoffeeHouse = new google.maps.Marker({
+    position: {lat: 51.525212, lng: -0.126469},
+    map: map,
+    title: 'Bloomsbury Coffee House',
+    icon: redIcon
+  });
+  
+  waitForClick(map, mildredsKingsCross);
+  waitForClick(map, skyGarden);
+  waitForClick(map, fitzBar);
+  waitForClick(map, bloomsburyCoffeeHouse);
+}
+
+function waitForClick(map, marker) {
+  var placeName = marker.getTitle();
+  var placeLongLat;
+  var placeCategory;
+  var placeRecommendation;
+
+  /* Iterate through stored recommendations to get details */
+  for (i = 0; i < recommendations.length; i++) {
+    var rec = recommendations[i]
+    if (rec[1] == placeName){
+      placeCategory = rec[0];
+      placeLongLat = rec[2];
+      placeRecommendation = rec[3];
+      break;
```

Comment:
```suggestion
    var recommendation = recommendations[i];
    if (recommendation[1] === placeName){
      placeCategory = recommendation[0];
      placeLongLat = recommendation[2];
      placeDescription = recommendation[3];
```
not sure, but "break" is kind of a bad practice(?)

Reply:
I think it's necessary here, to stop the for loop iterating over the other recommendations after the one it is looking for.